semcore/accordion/src/index.d.ts

```diff
@@ -92,4 +93,14 @@ declare const Accordion: IntergalacticAccordionComponent & {
   };
 };
 
+declare const wrapAccordion: <PropsExtending extends {}>(
+  wrapper: (
+    props: Intergalactic.InternalTypings.UntypeRefAndTag<
+      Intergalactic.InternalTypings.ComponentPropsNesting<IntergalacticAccordionComponent>
```

Comment:
`ComponentPropsNesting` should get a `TAG` in generic, not full `Intergalactic component` as I see...

Reply:
It's a quite powerful props extractor so it can accept a full `Interglactic.Component`